multisig/tests/scenario_go_test.rs

```diff
@@ -14,16 +14,19 @@ fn create_multiversx_to_ethereum_tx_batch_go() {
 }
 
 #[test]
+#[ignore] //Ignore for now
```

Comment:
will these ignored tests be fixed?

Reply:
Yep, will fix these tests in a following PR.